feature-libs/asm/styles/components/_customer-list.component.scss

```diff
@@ -8,7 +8,7 @@ $sapFioriContent_LabelColor: #6a6d70;
       min-height: 426px;
     }
 
-    @include forVersion(6) {
+    @include forVersion(6.1) {
```

Comment:
why this feature flag need to be changed?

Reply:
contacted with Yifeng in slack, it is for fixing typo

Reply:
This was discovered by us yesterday, and the previous writing was not correct

Reply:
This was previously omitted `.1` and should be `6.1`.